tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1107,7 +1108,7 @@ TEST (nnstreamer_capi_switch, dummy_01)
   status = ml_pipeline_destroy (handle);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  EXPECT_EQ (*count_sink, 3U);
+  EXPECT_GT (*count_sink, 0);
```

Comment:
consider to change EXPECT_LE (*count_sink, 3U);

Reply:
Good! I will add it. Thank you